### PR TITLE
support for clocker set start '10 minutes/hours ago', issue #27

### PR DESCRIPTION
**Pull request by nevilgeorge**
`clocker start`

`clocker status`
> elapsed time: 00:00:03

`clocker set start '10 minutes ago`'

`clocker status`
> elapsed time: 00:10:04

`clocker set start '1 hour ago`'

`clocker status`
> elapsed time: 1:00:04

### CODE REVIEW

File: bin/cmd.js [hunk 1]
```diff
@@ -254,7 +256,23 @@ else if (argv._[0] === 'set') {
             stamp = row.key.split('!')[1];
             prop = argv._[1];
             value = argv._.slice(2).join(' ');
-            set(stamp, prop, value);
+            terms = value.split(' ');
```
Comment (fnogatz) [inline bin/cmd.js]: I would try to not reinvent the wheel here: [parse-messy-time](https://github.com/substack/parse-messy-time) would fit perfectly.

```
> var parse = require('parse-messy-time')
> parse('now')
Mon Nov 16 2015 19:32:42 GMT+0100 (CET)
> parse('10 minutes ago')
Mon Nov 16 2015 19:22:42 GMT+0100 (CET)
> parse('10 min ago')
Mon Nov 16 2015 19:22:42 GMT+0100 (CET)
```

**Comment by fnogatz** [comment]
Hi Nevil!
Thank you for your contribution. It seems like a good solution for #27. Nevertheless I would prefer to use an existing parsing library like [parse-messy-time](https://github.com/substack/parse-messy-time) - just to separate the concerns.

**Comment by nevilgeorge** [comment]
Ah okay, I will use that and update the PR. Thanks for taking a look at it!

**Comment by nevilgeorge** [comment]
Just updated. Sorry for the delay.

**Comment by fnogatz** [comment]
Thanks for the update. I merged and changed the following things:
- Moved the `parseTime` magic into to the `updateDate` function - otherwise even `clocker set message '10 minutes ago'` would result in parsed time
- As a result, the parsing is applied for `clocker set STAMP start '10min ago'`, too

**Pull request merged**